### ISSUE TITLE
Crash in Advanced Content "Tierhandlung"

### ISSUE THREAD

**Issue by ThomasRitschel** [Pyrdacor/Ambermoon-Advanced#1]
When entering the room on the right in the "Tierhandlung", the game crashes (everytime) with message:
The given key "19" was not present in the dictionary.

**Comment by Pyrdacor** [comment]
Are you playing on an Amiga/Emulator or with Ambermoon.net?

**Comment by ThomasRitschel** [comment]
Ambermoon.net v0.01 deutsch on win10

**Comment by Pyrdacor** [comment]
I am not sure why this was happening, but it will definitely not happen for the new episode. My guess is that your savegame was started while the initial savegame was flawed.

When you enter that room, a merchant pops up. Your savegame lacks this merchant (index 19).

If you want I could repair your savegame. New games should not have the same issues. Even with the current state of episode 1.

**Comment by ThomasRitschel** [comment]
repairing the savegame would be great!

Thank you very much in advance
Thomas

Zitat von Pyrdacor ***@***.***>:

**Comment by ThomasRitschel** [comment]
Maybe I messed up the savegame because I was testing different versions of ambermoon.net to enter the crypt regarding the other issue I posted.Am 15.11.2022 21:18 schrieb Pyrdacor ***@***.***>:
I am not sure why this was happening, but it will definitely not happen for the new episode. My guess is that your savegame was started while the initial savegame was flawed.
When you enter that room, a merchant pops up. Your savegame lacks this merchant (index 19).
If you want I could repair your savegame. New games should not have the same issues. Even with the current state of episode 1.
